index.js

```diff
@@ -161,11 +162,11 @@ function createSeries (opts, archs, platforms) {
           })
         }
 
-        if (combination.platform === 'darwin') {
+        if (combination.platform === 'darwin' || combination.platform === 'mas') {
```

Comment:
As `util.js` is only included during testing. I'll write another isPlatformMac in `index.js` instead of requiring `test/util`?